frontend/src/app/components/MainInstallButton.js

```diff
@@ -49,11 +56,8 @@ export default class MainInstallButton extends React.Component {
       </div>
     );
   }
-
-  install() {
-    const { eventCategory, hasAddon } = this.props;
-    if (hasAddon) { return; }
-    this.setState({ isInstalling: true });
-    installAddon(eventCategory);
-  }
 }
+
+MainInstallButton.contextTypes = {
```

Comment:
This is how you get access to the store object. [More info](https://egghead.io/lessons/javascript-redux-passing-the-store-down-implicitly-via-context)